optashift-employee-rostering-gwtui/pom.xml

```diff
@@ -102,6 +102,27 @@
           <moduleName>org.optaplanner.openshift.employeerostering.gwtui.WorkerRosteringWebapp</moduleName>
         </configuration>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-resources-plugin</artifactId>
+        <version>2.3</version>
+      </plugin>
     </plugins>
+    <resources>
+      <resource>
+        <directory>src/main/java</directory>
+        <includes>
+          <include>**/*.html</include>
+          <include>**/*.css</include>
```

Comment:
Let's talk on bluejeans about this, I want to understand it.

Reply:
I think this is from using codeserver instead of devmode. All html and css files were in src/main/java instead of src/main/resources. If I run gwt:codeserver without this, these files will not be in the classes directory, causing errai to complain. Perhaps they should be moved to src/main/resources.

Reply:
+1 to move them instead of messing with the resources plugin

Let's talk on bluejeans anyway if you have some time in your morning (my afternoon), I want to understand this setup and it's not entirely clear to me.